patches/tModLoader/Terraria/ModLoader/ModBlockType.cs

```diff
@@ -11,8 +12,11 @@ public abstract class ModBlockType : ModTexturedType
 		/// <summary> The internal ID of this type of tile/wall. </summary>
 		public ushort Type { get; internal set; }
 
-		/// <summary> The default style of sound made when this tile/wall is hit. Defaults to null. </summary>
-		public SoundStyle? HitSound { get; set; } = null;
+		/// <summary>
+		/// The default style of sound made when this tile/wall is hit.<br></br>
```

Comment:
Are you not a certified _HTML programmer_?
```suggestion
		/// The default style of sound made when this tile/wall is hit.<br/>
```